xbmc/DynamicDll.cpp

```diff
@@ -83,6 +85,16 @@ bool DllDynamic::EnableDelayedUnload(bool bOnOff)
   return true;
 }
 
+bool DllDynamic::SetSystemLibrary(bool isSystem)
```

Comment:
This can be changed to return void as the return value is never checked and Coverity like to complain about ignoring return values.